readMind.html

```diff
@@ -1,22 +1,24 @@
 <script>
   var userName, name, userReady, message, answer;
 
-  userName = prompt("Hello, may I please have your name?");
-
-  name = userName
-    alert("Very nice to meet you, " + name + "!");
-
-  userReady = prompt("I am going to try and read your mind, " + name + ", is that okay?");
+  function Username(name, age) {
+    this.name = prompt("can I have your name?");
+    this.age = age;
+};
+   var name1 = new Username;
+    alert("Very nice to meet you, " + name1.name + "!");
+
+  userReady = prompt("I am going to try and read your mind, " + name1.name + ", is that okay?");
     if (userReady == "yes") {
-      message = "Alright, " + name + ", please think of a number.";
+      message = "Alright, " + name1.name + ", please think of a number.";
     } else {
-      message = "Very well, " + name + ", let me know when you are ready."
+      message = "Very well, " + name1.name + ", let me know when you are ready."
     };
```

Comment:
Your object constructor should have a method. Line 11 to 16 would make a great method.